spec/services/phone_number_capabilities_spec.rb

```diff
@@ -37,4 +37,32 @@
       expect(locality).to eq('Morocco')
     end
   end
+
+  describe 'list of unsupported area codes' do
+    it 'is up to date' do
+      unsupported_area_codes = {
+        '648' => 'American Samoa',
+        '264' => 'Anguilla',
+        '268' => 'Antigua and Barbuda',
+        '246' => 'Barbados',
+        '441' => 'Bermuda',
+        '345' => 'Cayman Islands',
+        '767' => 'Dominica',
+        '809' => 'Dominican Republic',
+        '473' => 'Grenada',
+        '671' => 'Guam',
+        '876' => 'Jamaica',
+        '664' => 'Montserrat',
+        '670' => 'Northern Mariana Islands',
+        '869' => 'Saint Kitts and Nevis',
+        '758' => 'Saint Lucia',
+        '784' => 'Saint Vincent Grenadines',
+        '868' => 'Trinidad and Tobago',
+        '649' => 'Turks and Caicos Islands',
+        '284' => 'British Virgin Islands',
+        '340' => 'United States Virgin Islands',
+      }
+      expect(PhoneNumberCapabilities::VOICE_UNSUPPORTED_US_AREA_CODES).to eq unsupported_area_codes
```

Comment:
I meant a UI spot check for a few of these? this is just copying a constant into two places?

Reply:
In this case, our goal is to make sure a specific set of area codes are flagged, and that only those codes are flagged, right? A few spot checks are not going to raise an error if we accidentally remove or add an area code that isn't tested.

Reply:
Alternatively, if the Twilio API provides this, we can fetch the list of unsupported area codes and compare it with our constant.